docker/api/container.py

```diff
@@ -193,12 +193,12 @@ def kill(self, container, signal=None):
 
     @utils.check_resource
     def logs(self, container, stdout=True, stderr=True, stream=False,
-             timestamps=False, tail='all', since=None):
+             timestamps=False, tail='all', since=None, follow=False):
         if utils.compare_version('1.11', self._version) >= 0:
             params = {'stderr': stderr and 1 or 0,
                       'stdout': stdout and 1 or 0,
                       'timestamps': timestamps and 1 or 0,
-                      'follow': stream and 1 or 0,
+                      'follow': follow and 1 or 0,
```

Comment:
I wonder if we should set `follow=None` and do:

```python

'follow': follow if follow is not None else int(stream)
```

to keep it backwards compatible